internal/daemon/daemon.go

```diff
@@ -578,9 +578,13 @@ func (d *Daemon) StartAPI(bootstrap bool, initConfig map[string]string, newConfi
 func (d *Daemon) addExtensionServers() error {
 	var networks []endpoints.Endpoint
 	for _, extensionServer := range d.extensionServers {
+		cert := extensionServer.Certificate
+		if cert == nil {
+			cert = d.ClusterCert()
+		}
```

Comment:
Should not be a blocker but the linter should normally complain that there is now newline after the `if` block. Something to follow up on our side :)

Reply:
This is a [custom linter in the LXD repo](https://github.com/canonical/lxd/blob/main/test/lint/newline-after-block.sh) but it's not present here.

Reply:
I will put in the new line here, going to steal that linter for the site manager repo too :slightly_smiling_face: